docs/resources/privatelink_aws.md

```diff
@@ -190,4 +194,12 @@ resource "cloudamqp_security_firewall" "firewall_settings" {
    ]
 }
 ```
+
 </details>
+
+## Changelog
+
+List of changes made to this resource for different versions.
+
+[v1.29.0](https://github.com/cloudamqp/terraform-provider-cloudamqp/releases/tag/v1.29.0) added
+configurable retries when reading PrivateLink information.
```

Comment:
This is a nice touch but I think it can be hard to maintain properly. I think it is better to mention it in and concentrate our effort on keeping https://github.com/cloudamqp/terraform-provider-cloudamqp/blob/main/CHANGELOG.md as good as possible. WDYT?

Reply:
True, tried it out here and liked it: https://github.com/cloudamqp/terraform-provider-cloudamqp/commit/a661e090bbab67e248a62206ba4cd0da753c82a9#diff-6575c05c10a8f7d9c1d7cec73bd49538df9c9ace86edd8c659a7365aa79bd799
Just to get rid of all version input in the page as much as possible and keep it listed last. But could also become a hassle to keep track on.

Reply:
Oh yeah I saw https://github.com/cloudamqp/terraform-provider-cloudamqp/pull/247/files#diff-9efdf0ce35a9a79cd4ff4aad80f648515a179242612ffa732cba9066aeb61c37R283 now

I think the information serves a purpose, like we add new capabilities to a resource, someone reading it may not use the latest version, so it is good for them to see what version they need to be using to take advantage of a certain thing.

I think we should specify that directly in the documentation next to the thing it applies to. "Available since v1.2.9". Would it be possible?

Reply:
Here are previous examples: 
As note boxes (looks better at registry and their markdown):
https://github.com/cloudamqp/terraform-provider-cloudamqp/blob/v1.28.0/docs/resources/plugin.md#cloudamqp_plugin

For each argument:
https://github.com/cloudamqp/terraform-provider-cloudamqp/blob/gcp-peering-retries/docs/resources/vpc_gcp_peering.md#argument-reference

Then perhaps due the latter part, but with `Available since v1.2.9` for each argument/attribute and skip the note part.

Reply:
Sounds good, will be lots of boxes otherwise.

Reply:
Wrong PR but just a concept. Example base on the discussion. 
https://github.com/cloudamqp/terraform-provider-cloudamqp/pull/247/commits/8def35362d9000e27ac84ff43e9a079bf2b73d16

Reply:
> Sounds good, will be lots of boxes otherwise.

heh yeah, hence why I tried to put them in the changelog section instead.

Reply:
> Wrong PR but just a concept. Example base on the discussion. [8def353](https://github.com/cloudamqp/terraform-provider-cloudamqp/commit/8def35362d9000e27ac84ff43e9a079bf2b73d16)

Looks good but still think we should concentrate on only one changelog